build/webpack/webpack.config.base.js

```diff
@@ -20,6 +22,63 @@ class AccessDependenciesPlugin {
   }
 }
 
+const defines = {
+  BUILDFLAG: ' '
```

Comment:
I'm confused how `BUILDFLAG(FOO)` ends up evaluating to true or false, but this otherwise lgtm

Reply:
+1
I don't see how it can work.

Reply:
I have this idea from @MarshallOfSound

Reply:
`BUILDFLAG(FOO)` -> ` (true)`, `BUILDFLAG` is just a wrapper for the flag, which is `true` or `false`